scheduler/replica_scheduler.go

```diff
@@ -678,14 +677,14 @@ func (rcs *ReplicaScheduler) GetDiskSchedulingInfo(disk longhorn.DiskSpec, diskS
 	return info, nil
 }
 
-func (rcs *ReplicaScheduler) CheckReplicasSizeExpansion(v *longhorn.Volume, oldSize, newSize int64) (err error) {
+func (rcs *ReplicaScheduler) CheckReplicasSizeExpansion(v *longhorn.Volume, oldSize, newSize int64) (multiError util.MultiError, err error) {
```

Comment:
Do we really need another wrap for the returned error? or using the returned error is good enough?

Reply:
The error contains different error messages, not sure if volume-scheduling-eerror should cover these different error messages. But I would prefer to let `multiError` focus on disk or space issue.

Reply:
I see, then I would suggest renaming the returned error name to `diskScheduleMultiError`. or similar to make it meaningful. WDYT?

Reply:
Sounds good!

Reply:
Updated. Thank you.